triaging.rst

```diff
@@ -3,193 +3,243 @@
-resource usage
-    Situations where too many resources (e.g. memory) are used.
-security
-    Issues that might have security implications.  If you think the issue
-    should not be made public, please report it to security@python.org instead.
-performance
-    Situations where too much time is necessary to complete the task.
-enhancement
-    Issues that propose the addition of new functionality, such as new
-    functions, classes, modules, or even new arguments for existing functions.
-    Also used for improvements in the documentation and test suite and for
-    other refactorings.
+Describes the type of issue.  If an issue does not fit within any
+specific type, please do not set a type.
+
++----------------+----------------------------------------------------------+
+|      Type      |                       Description                        |
++================+==========================================================+
+| behavior       | Unexpected behavior, result, or exception.  Most bugs    |
+|                | will have this type.                                     |
++----------------+----------------------------------------------------------+
+| compile error  | Errors reported by the compiler while compiling Python.  |
++----------------+----------------------------------------------------------+
+| crash          | Hard crashes of the Python interpreter -- possibly with  |
+|                | a core dump or a Windows error box.                      |
++----------------+----------------------------------------------------------+
+| enhancement    | Issues that propose the addition of new functionality,   |
+|                | such as new functions, classes, modules, or even new     |
+|                | arguments for existing functions. Also used for          |
+|                | improvements in the documentation, test suite and        |
+|                | other refactorings. A good place to discuss enhancements |
+|                | prior to filing an issue is **python-ideas** mailing     |
+|                | list.                                                    |
++----------------+----------------------------------------------------------+
+| performance    | Situations where too much time is necessary to complete  |
+|                | the task. For example, a common task now takes           |
+|                | significantly longer to complete.                        |
++----------------+----------------------------------------------------------+
+| resource usage | Situations where too many resources (e.g. memory) are    |
+|                | used.                                                    |
++----------------+----------------------------------------------------------+
+| security       | Issues that might have security implications. Report     |
+|                | security vulnerabilities which should not be made public |
+|                | to security@python.org.                                  |
```

Comment:
Should probably link to https://www.python.org/news/security/ as it has e.g. PGP key to use.